src/pages/lib/graphqlapi/subscribe-data/q/platform/[platform].mdx

```diff
@@ -1,11 +1,11 @@
 export const meta = {
   title: `Subscribe to real-time events`,
-  description: `Learn more about how to observe to data changes using subscriptions in Amplify.`
+  description: `Learn more about how to observe data changes using subscriptions in Amplify.`
 };
 
 <InlineFilter filters={["js", "react-native"]}>
 
-In this guide, we outline the benefits of enabling real-time data integrations and how to set up and filter these subscriptions. We will also cover how to unsubscribe from subscriptions when needed.
+In this guide, we will outline the benefits of enabling real-time data integrations and how to set up and filter these subscriptions. We will also cover how to unsubscribe from subscriptions.
 
 Before you begin, you will need:
 
```

Comment:
If Connect guide is not ready - the JS link below will need to be updated to getting started page.